lib/server/services/run/RunService.js

```diff
@@ -301,6 +377,85 @@ class RunService {
             .findAll(queryBuilder))
             .map((row) => ({ l3Level: row.get('l3Level'), dipoleLevel: row.get('dipoleLevel') }));
     }
+
+    // eslint-disable-next-line valid-jsdoc
```

Comment:
Why disabled?

Reply:
For some reason it do not accept the lambda function description

Reply:
what about `function<Error, void>`

Reply:
For my IDE it do not work?